metrics-graphite/src/main/java/com/codahale/metrics/graphite/GraphiteReporter.java

```diff
@@ -15,7 +26,21 @@
 import java.util.concurrent.ScheduledExecutorService;
 import java.util.concurrent.TimeUnit;
 
-import static com.codahale.metrics.MetricAttribute.*;
+import static com.codahale.metrics.MetricAttribute.COUNT;
```

Comment:
Shouldn't static imports typically be listed before the normal imports?

Reply:
I agree that they should be listed before the normal imports, that can be tackled in a different PR. This is just setting things up to use the same configuration as the Dropwizard project.

http://checkstyle.sourceforge.net/config_imports.html#CustomImportOrder